OfficeToPDF/ExcelConverter.cs

```diff
@@ -410,6 +408,14 @@ public static new int Convert(String inputFile, String outputFile, Hashtable opt
 
                 workbook.SaveAs(tmpFile, fmt, Type.Missing, Type.Missing, Type.Missing, false, XlSaveAsAccessMode.xlNoChange, Type.Missing, false, Type.Missing, Type.Missing, Type.Missing);
 
+                // Fit to to one page wide.
+                if (fitToPageWidth)
+                {
+                    templatePageSetup["FitToPagesWide"] = 1;
+                    templatePageSetup["FitToPagesTall"] = false;

```

Comment:
Q: shouldn't this be a seperate flag?

Reply:
In zoom convo we decided that we would have only 1 flag for the entire functionality.  Still open to change this though.

Reply:
Decided to keep this as one logical flag.